crates/bindings-csharp/Codegen.Tests/fixtures/server/Lib.cs

```diff
@@ -39,12 +39,13 @@ public partial record CustomTaggedEnum
     : SpacetimeDB.TaggedEnum<(int IntVariant, string StringVariant)>;
 
 [SpacetimeDB.Table]
-public partial class PrivateTable { }
+public partial struct PrivateTable { }
```

Comment:
Is this change necessary? It had a secondary purpose of checking that `class` tables work as well. Admittedly, that's not very clear, so if you want to instead add a class-based table separately, that would be fine too.

Reply:
Ah hmm, I'll have to relax other constraints then.

Reply:
Ah heh then test catches an issue as expected.

Reply:
(still needs reverted so we test class-based tables)